orca-echo/src/test/groovy/com/netflix/spinnaker/orca/echo/spring/EchoNotifyingExecutionListenerSpec.groovy

```diff
@@ -136,8 +136,8 @@ class EchoNotifyingExecutionListenerSpec extends Specification {
 
     then:
     pipeline.notifications.size() == 2
-    pipeline.notifications.when == [["pipeline.started", "pipeline.completed"], ["pipeline.failed"]]
-    pipeline.notifications.extraField == ["extra", null]
+    pipeline.notifications.when.containsAll(["pipeline.started", "pipeline.completed"], ["pipeline.failed"])
+    pipeline.notifications.extraField.containsAll("extra", null)
```

Comment:
Seemed to be an inconsistent ordering behavior here but as order doesn't matter to the use-case I changed the assertion.